src/probe_modules/module_bacnet.c

```diff
@@ -111,11 +113,12 @@ int bacnet_validate_packet(const struct ip *ip_hdr, uint32_t len,
 			   uint32_t *src_ip, uint32_t *validation,
 			   const struct port_conf *ports)
 {
-	// this will reject packets that aren't UDP or ICMP and fully process ICMP
-	// packets
-	if (udp_do_validate_packet(ip_hdr, len, src_ip, validation, num_ports,
-				   SRC_PORT_VALIDATION,
-				   ports) == PACKET_INVALID) {
+	// this will reject packets that aren't UDP or ICMP and fully process ICMP packets
+	bool should_validate_source_port = true; // default to validating source port
```

Comment:
This feels like it should be set globally somehow, not with the constant hidden in the function?